server/src/test/java/com/cloud/configuration/ConfigurationManagerTest.java

```diff
@@ -166,6 +168,8 @@
 
     VlanVO vlan = new VlanVO(Vlan.VlanType.VirtualNetwork, "vlantag", "vlangateway", "vlannetmask", 1L, "iprange", 1L, 1L, null, null, null);
 
+    private static final String MAXIMUM_IOPS_ALLOWED = "3600";
```

Comment:
Maybe rename this to MAXIMUM_LENGTH_ALLOWED as it's a time which we specify here?

Reply:
imho: MAXIMUM_DURATION_ALLOWED